### ISSUE TITLE
run.py line 46 says "test.sh" but there is no such file

### ISSUE THREAD

**Issue by ShunyaWatanabe** [nabilr/netlab#1]
Should it be like that?

Also, I would really appreciate if there is a better and more detailed explanation on how to run these tests. I'm so not understanding what's going on in these codes.

**Comment by nabilr** [comment]
details steps were given readme documents.  Please refer it. setup.sh is creating test.sh file

**Comment by ShunyaWatanabe** [comment]
I read it and I ran the commands but it doesn't work. 

after "bash setup.sh 2 nc", nothing happens.

**Comment by ShunyaWatanabe** [comment]
ok. I figured that I had to wait for a while for everything to get compiled or something.

Am I also supposed to bring the executable files (./FTPclient and ./FTPserver) in the same directory as these files?

**Comment by nabilr** [comment]
Yes True... You have to CopY (./FTPclient and ./FTPserve in same directory

Please pull the latest code changes.
